src/static/riot/results.tag

```diff
@@ -184,6 +170,21 @@
             // Template stuff
             $('.datepicker').calendar({
                 type: 'date',
+                formatter: {
```

Comment:
can we use a library to format date? do we already have a js datetime lib? https://github.com/moment/luxon

`dt.toLocaleString()`

Then when sending to server you can `dt.toISO()` then we don't need special formatter on the other end for search?

Reply:
Using luxon now to send in ISO format.

Reply:
```formatter: {
                    date: function (date, settings) {
                        dt = luxon.DateTime.fromJSDate(date).toISO()
                        return dt
                    }
                },```